signup.php

```diff
@@ -0,0 +1,43 @@
+
+
```

Comment:
Related to my comment at the top of this PR, this probably looks harmless, but this is technically sending two "new line" characters to the browser before entering the PHP block. This means it's too late to do certain things such as redirect the page. Normally whitespace doesn't matter, but it can at the very beginning and ending of files. So I'd suggest always making `<?php` the very first thing, and this is also why it's best to not have a closing `?>` tag at the end of a file (so you don't accidentally send unintentional whitespace to the browser)